.github/workflows/pr-title-update.yml

```diff
@@ -0,0 +1,25 @@
+name: Update PR Title If Target Branch Is A Version Branch
+
+on:
+  pull_request_target:
+    types: [opened]
+
+jobs:
+  update-title:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update PR title
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
You can probably use https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#permissions instead of the token, see e.g. `.github/workflows/maintenance_bot.yaml`

Reply:
We have to set permissions but in `maintenance_bot.yaml` it is using `actions/github-script` that will automatically get it.

Reply:
`actions/labeler`, also used in that workflow, needs the permissions: https://github.com/actions/labeler?tab=readme-ov-file#permissions

Reply:
Sorry, I forgot to reply!
Here for using GitHub CLI, we have to use GITHUB_TOKEN:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/using-github-cli-in-workflows